EDU-998: update CLI docs (#2267)
temporalio/documentation: docs-src/cli/task-queue/get-build-id-reachability.md

> ---

Reviewer: Add the new `temporal task-queue` subcommands (`get-build-id-reachability`, `get-build-ids`, and `update-build-ids`) to the `task-queue.json` config file.

id: get-build-id-reachability
title: temporal task-queue get-build-id-reachability
sidebar_label: get-build-id-reachability

> description: Retrieves information about the reachability of Build IDs on one or more Task Queues

Reviewer: ```suggestion
description: Retrieves information about the reachability of Build IDs on one or more Task Queues.
```

tags:
  - cli reference
  - temporal cli
  - task queue
  - command-line-interface-cli
---

> Determines if Build IDs may be used for for new, existing, or closed Workflows.
> Both the `--build-id` and `--task-queue` flags may be specified multiple times.
> Reachability for the provided Build IDs will be checked against all task queues if one isn't provided.

Reviewer: ```suggestion
Determines whether Build IDs can be used for for new, existing, or closed Workflows.
Both the `--build-id` and `--task-queue` options can be specified multiple times.
If a Task Queue isn't provided, reachability for the provided Build IDs is checked against all Task Queues.
```
